mix.exs

```diff
@@ -22,9 +22,9 @@ defmodule HTTPoison.Mixfile do
 
   defp deps do
     [
-      {:hackney, "~> 1.6.0"},
+      {:hackney, "~> 1.6.0 and >= 1.6.3"},
```

Comment:
`~> 1.6.3` would not update to 1.7.0 when it comes out. I was trying to replicate the old `~> 1.6` requirement